pkg/program/shell/runtime.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/elves/elvish/pkg/daemon"
 	"github.com/elves/elvish/pkg/eval"
 	daemonmod "github.com/elves/elvish/pkg/eval/daemon"
+	elvish_math "github.com/elves/elvish/pkg/eval/math"
```

Comment:
Do not use underscores in identifiers. Follow the pattern above and call the import alias `mathmod`.

Reply:
Okay

Reply:
Wouldn't it be more consistent to use snake-case to mimic the var naming convention; i.e., `mathMod`?